input-events/idlharness.html

```diff
@@ -0,0 +1,40 @@
+<title>Input Event IDL tests</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/WebIDLParser.js"></script>
+<script src="/resources/idlharness.js"></script>
+
+<pre id="untested_idl">
+interface InputEventInit {
+};
+
+interface UIEvent {
+};
+
+interface StaticRange {
+};
+</pre>
+
+<pre id='idl'>
+[Constructor(DOMString type, optional InputEventInit eventInitDict)]
+interface InputEvent : UIEvent {
```

Comment:
This doesn't match https://w3c.github.io/uievents/#events-inputevents

In order to make it easy to keep these tests up to date, it'd be good to copy spec IDL verbatim, and link to where it was copied from.